### PR TITLE
Open source singleton test

### PR DESCRIPTION
**Pull request by Ankur19**

### CODE REVIEW

File: feature/interface/singleton/ate_tests/singleton_test/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# RT-5.1: Singleton Interface
+
+## Summary
+
+Singleton L3 interface (non-LAG) is supported on DUT.
+
+## Procedure
```
Comment (liulk) [inline feature/interface/singleton/ate_tests/singleton_test/README.md:7]: I updated the test plan. Please use the updated version here.

**Review by liulk — CHANGES_REQUESTED** [review]

File: feature/interface/singleton/ate_tests/singleton_test/README.md (hunk 1 above)
Comment (Ankur19) [reply feature/interface/singleton/ate_tests/singleton_test/README.md:7]: Done!

File: feature/interface/singleton/ate_tests/singleton_test/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+*   For each port speed and breakout port configuration that need to be tested,
```
Comment (liulk) [inline feature/interface/singleton/ate_tests/singleton_test/README.md:9]: Dedent everything by one level? Namely, "for each port speed" will not be a bullet point, and "Configure ..." will be level 1, etc.

Comment (Ankur19) [reply feature/interface/singleton/ate_tests/singleton_test/README.md]: Done!

**Review by liulk — APPROVED** [review]

**Pull request merged**